### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.3][IntelliJ][Wipro]:  Table commands are not working for tables present in Run on Web App Dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1493]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select "Use Existing web app" radio button.
6. In Run on Web App dialog, press tab to navigate tables then try to access the table cells by using table commands(CTRL+Alt+Arrow keys) then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
By using below table command keys user should be able to access the table and NVDA should announced the Value present in the table cell then row position header name and column position
 Ctrl + Alt + Right Arrow-> Cell to right
Ctrl + Alt +Left Arrow->Cell to left
Ctrl + Alt +Down Arrow-> Cell below
Ctrl + Alt +Up Arrow->Cell above

**Actual Result:**
**NVDA Behavior:**
NVDA is not announcing anything by using table command keys and table keys are not working
**Jaws Behavior:** Jaws is announcing as expected
Column 1, Row 1
Name
webapp-180112171944
JDK
1.8
Column 2
Web container
TOMCAT 8.5
Column 3
Resource group
adobe-rg
Column 4

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![table keys are not working in run on web app](https://user-images.githubusercontent.com/22956130/36136076-a7d9c442-10b4-11e8-8232-0e3cf5b8d0d2.jpg)
![run on web app-table](https://user-images.githubusercontent.com/22956130/36136088-b3358588-10b4-11e8-8fee-c77fbe637617.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749949
